static/js/settings_org.js

```diff
@@ -648,6 +648,32 @@ export function discard_property_element_changes(elem, for_realm_default_setting
     update_dependent_subsettings(property_name);
 }
 
+export function deactivate_organization(e) {
+    e.preventDefault();
+    e.stopPropagation();
+
+    function do_deactivate_realm() {
+        channel.post({
+            url: "/json/realm/deactivate",
+            error(xhr) {
+                ui_report.error($t_html({defaultMessage: "Failed"}), xhr, $("#dialog_error"));
+            },
+        });
+    }
+
+    const html_body = render_settings_deactivate_realm_modal();
+
+    dialog_widget.launch({
```

Comment:
The `confirm_dialog` by default uses `close_on_submit: true`, which is not exactly what we wanted. Since `confirm_dialog` internally calls `dialog_widget`, I preferred to use `dialog_widget` instead of `confirm_dialog` so that I could set `close_on_submit: false`, which allows errors to be shown within the modal itself.

Reply:
Ok, sounds good for now.

Reply:
Ok sounds good for now.